tests/pipelines/eks/awscli-cl2-load-with-addons.yaml

```diff
@@ -28,34 +30,83 @@ spec:
     taskRef:
       kind: Task
       name:  slack-notification
+  - name: create-cluster-service-role
+    params:
+    - name: stack-name
+      value: $(params.cluster-name)-service-role
+    - name: role-cfn-url
+      value: $(params.service-role-cfn-url)
+    - name: role-name
+      value: $(params.cluster-name)-service-role
+    runAfter:
+    - slack-notification
+    taskRef:
+      kind: Task
+      name: awscli-role-create
+  - name: create-cluster-node-role
+    params:
+    - name: stack-name
+      value: $(params.cluster-name)-node-role
+    - name: role-cfn-url
+      value: $(params.node-role-cfn-url)
+    - name: role-name
+      value: $(params.cluster-name)-node-role
+    runAfter:
+    - slack-notification
+    taskRef:
+      kind: Task
+      name: awscli-role-create
   - name: create-eks-cluster
     params:
     - name: cluster-name
       value: $(params.cluster-name)
-    - name: servicerole
-      value: $(params.servicerole)
+    - name: service-role-name
+      value: $(params.cluster-name)-service-role
     - name: endpoint
       value: $(params.endpoint)
     runAfter:
-    - slack-notification
+    - create-cluster-node-role
```

Comment:
this should be `create-cluster-service-role` as we need service role for `create-cluster` task

Reply:
we can have both..we can do the stack setup in parallel to save some time. :)

Reply:
Yes, we can have both too. As long as we have service role created before we can save race condition failure mode. Discussed offline on this.